Makefile

```diff
@@ -28,7 +28,7 @@ release: \
 	_output/release/bootkube.tar.gz \
 
 check:
-	@find . -name vendor -prune -o -name '*.go' -exec gofmt -s -d {} +
+	@gofmt -s -d $$(find . -name '*.go' -print -o -name vendor -prune); if [ "$$?" = "0" ]; then exit 1; fi
```

Comment:
we could use `@gofmt -s -d $$(find . -name '*.go' -print -o -name vendor -prune) && exit 1`